luma/oled/device.py

```diff
@@ -374,6 +379,10 @@ def display(self, image):
 
         if self.framebuffer.redraw_required(image):
             left, top, right, bottom = self.framebuffer.bounding_box
+            _,_,w,h = self.bounding_box
```

Comment:
here are `h-offset` and `v-offset` params used elsewhere which we should emulate here. See: https://github.com/rm-hull/luma.lcd/commit/116c09abec7e3ab388699b2869ce9f07fd32a50a